.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       script: make docker
     - name: 3d smFISH Notebook
       if: type = push and branch =~ /^(master|merge)/
-      script: make install-dev run__notebooks/py/3d_smFISH.py
+      script: make install-dev && ipython docs/source/_static/data_processing_examples/3d_smFISH.py
```

Comment:
Do you care that we don't run this with the `run-notebooks` rule?  I suspect we want to because run-notebooks is run as part of the release process.